.clang-tidy

```diff
@@ -0,0 +1,44 @@
+---
+Checks: >
+  clang-diagnostic-*,
+  clang-analyzer-*,
+  -*,
+  performance-faster-string-find,
+  performance-for-range-copy,
+
+WarningsAsErrors: '*'
+HeaderFilterRegex: '/(?!external)/.*'
+AnalyzeTemporaryDtors: false
+FormatStyle:     none
+CheckOptions:    
```

Comment:
I propose to remove "CheckOptions:".
Any way default values for "CheckOptions:" were used